kokoro/build_binaries_oak_containers.sh

```diff
@@ -17,28 +16,37 @@ cd "$(dirname "$0")/.."
 ./scripts/docker_pull
 ./scripts/docker_run nix develop .#containers --command just kokoro_oak_containers
 
-mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
+mkdir -p "${KOKORO_ARTIFACTS_DIR}/test_logs/"
 cp --preserve=timestamps \
     ./target/nextest/default/*.xml \
     "${KOKORO_ARTIFACTS_DIR}/test_logs/" || true
 
-mkdir -p "$KOKORO_ARTIFACTS_DIR/binaries/"
+mkdir -p "${KOKORO_ARTIFACTS_DIR}/binaries/"
 
-# Store the git commit hash in the name of an empty file, so that it can be efficiently found via a glob.
-touch "$KOKORO_ARTIFACTS_DIR/binaries/git_commit_${KOKORO_GIT_COMMIT_oak:?}"
+# Store the git commit hash in the name of an empty file, so that it can
+# be efficiently found via a glob.
+touch "${KOKORO_ARTIFACTS_DIR}/binaries/git_commit_${KOKORO_GIT_COMMIT_oak:?}"
 
 # Copy the generated binaries to Placer. The timestamps are used to convey
 # the creation time.
-export GENERATED_BINARIES=(
+readonly generated_binaries=(
     ./target/stage1.cpio
-    ./oak_containers_kernel/target/vmlinux
     ./oak_containers_kernel/target/bzImage
     ./oak_containers_system_image/target/image.tar.xz
     ./oak_containers_hello_world_container/target/oak_container_example_oci_filesystem_bundle.tar
     ./oak_functions_containers_container/target/oak_functions_container_oci_filesystem_bundle.tar
 )
-cp --preserve=timestamps \
-    "${GENERATED_BINARIES[@]}" \
-    "${KOKORO_ARTIFACTS_DIR}/binaries/"
+readonly binary_names=(
+    oak_containers_stage1
+    oak_containers_kernel
+    oak_containers_system_image
+    oak_containers_hello_world_container # TBD
+    oak_containers_functions # TBD
```

Comment:
The first four binary names are no-brainers, but the fifth is awkward. If you agree with `oak_containers_functions`, should the source folder be renamed?